### PR TITLE
heartbeat support

### PR DESCRIPTION
**Pull request by mente**
AMQP extension [supports](https://github.com/pdezwart/php-amqp/commit/4918f2e19c8ae5058c3fab535621fad963ed5438) heartbeat since 1.6.0beta4. So why not support it as well?

### CODE REVIEW

**Comment by omansour** [comment]
maybe we should add a constraint about the extension version in the composer.json

**Comment by mente** [comment]
Why? You can live w/o heartbeat functionality. And 1.6.0 was released like 2 weeks ago only

File: src/AmqpBundle/DependencyInjection/M6WebAmqpExtension.php [hunk 1]
```diff
@@ -43,21 +43,23 @@ public function load(array $configs, ContainerBuilder $container)
     protected function loadConnections(ContainerBuilder $container, array $config)
     {
         foreach ($config['connections'] as $key => $connection) {
-            $connexionDefinition = new Definition($connection['class']);
-            $connexionDefinition->addMethodCall('setHost ', [$connection['host']])
+            $connectionDefinition = new Definition($connection['class']);
+            $connectionDefinition->addMethodCall('setHost ', [$connection['host']])
```
Comment (omansour) [inline src/AmqpBundle/DependencyInjection/M6WebAmqpExtension.php]: space after setHost

**Comment by omansour** [comment]
no problem to add the heartbeat option with an extension prior to 1.6 ?

**Comment by mente** [comment]
Nope, I've originally tested with 1.4.0 and 1.6.0. Works fine

File: src/AmqpBundle/DependencyInjection/M6WebAmqpExtension.php (hunk 1 above)
Comment (mente) [inline src/AmqpBundle/DependencyInjection/M6WebAmqpExtension.php]: it was there before :)

Comment (fabdsp) [inline src/AmqpBundle/DependencyInjection/M6WebAmqpExtension.php]: Maybe time to fix it ! ;-)

**Comment by fabdsp** [comment]
:+1:

**Comment by t-geindre** [comment]
:+1:

**Comment by fabdsp** [comment]
v1.9.0

**Pull request merged**